dashboard/models.py

```diff
@@ -140,6 +140,23 @@ def __str__(self):
     class Meta:
         managed = False
         db_table = 'course_view_option'
+    
+    def json(self):
+        """Format the json output that we want for this record
+        
+        This should be of the format canvas_id : {options}
+        :return: JSON formatted CourseViewOption
+        :rtype: Dict
+        """
+
+        return {
+            self.course.canvas_id: {
+                'cn': self.course.name, 
+                'fa': int(self.show_files_accessed),
+                'ap': int(self.show_assignment_planning),
+                'gd': int(self.show_grade_distribution),
```

Comment:
will the json feed will have the abbreviated properties like `cn`,`fa`? I feel we should have clear names for readability purposes

Reply:
I have it abbreviated for bandwidth cost savings. We're charged by KB and also it would be a quicker download for the user. This will be 100% machine read. It lowers the per course size from about 100 bytes to like 50.

Reply:
This isn't a *huge* savings right now (maybe a few dollars) but if we supported a lot of courses it would be could be a lot. Though we might have to think of a better solution at that time.

Reply:
Look at #332 it's an example of the JSON generated by this.